redacted-compiler-plugin-gradle/src/main/kotlin/dev/zacsweers/redacted/gradle/RedactedGradleSubplugin.kt

```diff
@@ -48,27 +48,40 @@ class RedactedGradleSubplugin : KotlinCompilerPluginSupportPlugin {
     val annotation = extension.redactedAnnotation
 
     // Default annotation is used, so add it as a dependency
+    // Note only multiplatform, jvm/android, and js are supported. Anyone else is on their own.
     if (annotation.get() == DEFAULT_ANNOTATION) {
-      val isMultiplatform = project.plugins.hasPlugin("org.jetbrains.kotlin.multiplatform")
-      if (isMultiplatform) {
-        val sourceSets =
-            project.extensions.getByType(KotlinMultiplatformExtension::class.java).sourceSets
-        val sourceSet = (sourceSets.getByName("commonMain") as DefaultKotlinSourceSet)
-        project
-            .configurations
-            .getByName(sourceSet.apiConfigurationName)
-            .dependencies
-            .add(
-                project.dependencies.create(
-                    "dev.zacsweers.redacted:redacted-compiler-plugin-annotations:$VERSION"))
-      } else {
-        project
-            .configurations
-            .getByName("api")
-            .dependencies
-            .add(
-                project.dependencies.create(
-                    "dev.zacsweers.redacted:redacted-compiler-plugin-annotations-jvm:$VERSION"))
+      when {
+        project.plugins.hasPlugin("org.jetbrains.kotlin.multiplatform") -> {
+          val sourceSets =
+              project.extensions.getByType(KotlinMultiplatformExtension::class.java).sourceSets
+          val sourceSet = (sourceSets.getByName("commonMain") as DefaultKotlinSourceSet)
+          project
+              .configurations
+              .getByName(sourceSet.apiConfigurationName)
+              .dependencies
+              .add(
+                  project.dependencies.create(
+                      "dev.zacsweers.redacted:redacted-compiler-plugin-annotations:$VERSION"))
+        }
+        project.plugins.hasPlugin("org.jetbrains.kotlin.js") -> {
+          project
+              .configurations
+              .getByName("api")
+              .dependencies
+              .add(
+                  project.dependencies.create(
+                      "dev.zacsweers.redacted:redacted-compiler-plugin-annotations-js:$VERSION"))
```

Comment:
Just tested this in a fresh JS project and all is working great. Though I did some more testing and found that the following also works in a JS project:
```kotlin
api("dev.zacsweers.redacted:redacted-compiler-plugin-annotations:0.10.0-RC1")
```
So I guess this could be simplified since the only exception would be the configuration name for multiplatform projects.

Reply:
Yep I think so too, I'll update with that

Reply:
Done and released in 0.10.0